app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -75,6 +80,19 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
         initSession()
     }
 
+    @SuppressLint("RestrictedApi")
+    override fun onResume() {
+        super.onResume()
+        if (session.isYoutubeTV) {
+            // Send key events on the UI thread to clear webview grey screen, see #1865
+            GlobalScope.launch(Dispatchers.Main) {
+                delay(50)
+                activity?.dispatchKeyEvent(KeyEvent(KeyEvent.ACTION_DOWN, KeyEvent.KEYCODE_DPAD_RIGHT))
```

Comment:
Nit: I think it would be good to include a comment with a few of the things you tried to get this to work.  I know the information is in the comment, but a brief list helps to contextualize why we're doing this.